tasks/set_up_rhaptos_site.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: set up localFS PDF folders
+  shell: "./bin/instance run scripts/set_up_localfs_for_pdfs.py {% for item in archive_exports_directories|default(default_archive_exports_directories) %}pdfs{{ loop.index }}:/var/www/{{ item }} {% endfor %}"
+  args:
+    chdir: "/var/lib/cnx/cnx-buildout"
+
+- name: set up rhaptos print host
+  shell: "echo -c \"app.plone.rhaptos_print._host = 'http://{{ frontend_domain }}'; import transaction; transaction.commit()\" | ./bin/instance debug"
+  args:
+    chdir: "/var/lib/cnx/cnx-buildout"
+
+- name: set up collections allowed content types
+  shell: "echo -c \"app.plone.portal_types.Collection.allowed_content_types += ('UnifiedFile',); import transaction; transaction.commit()\" | ./bin/instance debug"
+  args:
```

Comment:
Not idempotent, so we _might_ end up w/ duplicates, but this is only run on initial setup, so probably not.
A fix would be to use something like:
```app.plone.portal_types.Collection.allowed_content_types += [('UnifiedFile',),()]['UnifiedFile' in app.plone.portal_types.Collection.allowed_content_types]```
(This uses the False==0, True==1 hack to index into a two-value list to only make the += a noop if the value is already present)